pom.xml

```diff
@@ -85,7 +85,7 @@
         <dependency>
             <groupId>org.jboss.aerogear.unifiedpush</groupId>
             <artifactId>unifiedpush-push-model</artifactId>
-            <version>1.1.0-alpha.1</version>
+            <version>1.1.0-alpha.2-SNAPSHOT</version>
```

Comment:
@matzew Ah but that only happens for the version of the project itself, no ? Here we are speaking of a dependency